bootstrap.sh

```diff
@@ -1,27 +1,45 @@
 #!/usr/bin/env bash
 # Usage: ./bootstrap.sh
-# This script will install swiftformat, link git hooks and install required ruby gems.
+# This script will install Mint and bootstrap its dependencies, link git hooks and install required ruby gems.
 # You should have homebrew installed.
 # If you get `zsh: permission denied: ./bootstrap.sh` error, please run `chmod +x bootstrap.sh` first
 
-# Install swiftformat 0.45.6
-brew unlink swiftformat
-brew tap-new getstream/ios
-brew extract --version 0.45.6 swiftformat getstream/ios
-brew reinstall getstream/ios/swiftformat@0.45.6
-brew untap getstream/ios
+export SWIFTFORMAT_VERSION=0.45.6
+export SWIFTLINT_VERSION=0.41.0 # Also used by danger-swiftlint
+
+# Check if Homebrew is installed
+if [[ $(command -v brew) == "" ]]; then
+    echo "Homebrew not installed. Please install."
+    exit 1
+fi
+
+echo
+echo "👉 Generate mintfile"
+echo "# Warning - this file is generated using bootstrap.sh, don't edit it directly." > mintfile
+echo "nicklockwood/SwiftFormat@$SWIFTFORMAT_VERSION" >> mintfile
+echo "realm/SwiftLint@$SWIFTLINT_VERSION" >> mintfile
+cat mintfile
```

Comment:
Why don't we just check-in the mintfile to repo?

Reply:
I'm just trying something out, will probably end up having it checked out to the repo directly...

Reply:
Unfortunately, Danger usesSWIFTLINT_VERSION env variable for configuration and we have to use it in Mint file, too. I wanted to have a single source of truth, so generating Mint file like this seems easier than parsing it and extracting the value from it.

Reply:
```sh
cat Mintfile | grep SwiftLint | awk -F@ '{print $2}'
```

Reply:
IMHO Mintfile generation sounds fishy.
I would expect it to be clear declaration

Reply:
OK, I'll get it from the mintfile. Fixed in 08a0e2d